test/sagemaker_tests/pytorch/inference/integration/__init__.py

```diff
@@ -39,6 +40,10 @@
 resnet_neuron_script = os.path.join(model_neuron_dir, code_sub_dir, 'resnet18.py')
 resnet_neuron_input = os.path.join(model_neuron_dir, 'cat.jpg')
 resnet_neuron_image_list = os.path.join(model_neuron_dir, 'imagenet1000_clsidx_to_labels.txt')
+model_neuronx_dir = os.path.join(resnet_path, neuronx_sub_dir)
```

Comment:
are these file duplicates? if so, host it in a common folder name and refer the path here. We do not want duplicate files

Reply:
Removed the image and the labels.txt file, the other two files are unique to neuronx